stdlib/3.4/enum.pyi

```diff
@@ -1,12 +1,14 @@
-# FIXME: Stub incomplete, ommissions include:
-# * the metaclass
-# * _sunder_ methods with their transformations
-
+import abc
 import sys
-from typing import List, Any, TypeVar, Union
+from typing import List, Any, TypeVar, Union, Iterable, Iterator, TypeVar, Generic, Type
+
+_T = TypeVar('_T', bound=Enum)
 
-class Enum:
-    def __new__(cls, value: Any) -> None: ...
+class EnumMeta(abc.ABCMeta, Iterable[Enum]):
```

Comment:
In the enum.py code it inherits from `type`, not `ABCMeta`. Is there a reason to use ABCMeta here?